src/caretogether-pwa/src/Families/FamilyScreen.tsx

```diff
@@ -386,6 +402,23 @@ export function FamilyScreen() {
               </Grid>}
           </Grid>
           <Grid container spacing={0}>
+			<Grid item xs={12}>
```

Comment:
Put this higher up on the Family screen, above the row of "Incomplete/Completed/Documents" headers in the grid.

Reply:
Location updated